core/js/pokemon.maps.js

```diff
@@ -46,6 +51,39 @@ function initMap() {
 		$.getJSON( 'core/json/defaultstyle.json', function( data ) {
 			map.set('styles', data);
 		});
+
+		$.ajax({
+			'async': true,
+			'type': "GET",
+			'global': false,
+			'dataType': 'json',
+			'url': "core/process/aru.php",
+			'data': {
+				'request': "",
+				'target': 'arrange_url',
+				'method': 'method_target',
+				'type': 'pokemon_coordinates_area'
+			}
+		}).done(function(coordinates) {
+			getArea(coordinates);
+		});
+		
+		if (navigator.geolocation) {
+			navigator.geolocation.getCurrentPosition(function(position) {
+				var pos = {
+					lat: position.coords.latitude,
+					lng: position.coords.longitude
+				};
+
+				coordinate_area();
+
+				if (position.coords.latitude <= max_latitude && position.coords.latitude >= min_latitude) {
+					if (position.coords.longitude <= max_longitude && position.coords.longitude >= min_longitude) {
+						map.setCenter(pos);
+					}
+				}
+			});
+		}
```

Comment:
The whole block from `if (navigator.geolocation) {` to this line have to be in `.done(function(coordinates) {`.